### PR TITLE
BUGFIX: TMA-386 LCM2 Release brick: bad performance

### PR DESCRIPTION
**Pull request by vhtien**

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/523/ : SUCCESS in 15s
- https://ci.intgdc.com/job/gooddata-ruby-tests-zuul/462/ : SUCCESS in 44m 52s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/199/ : FAILURE in 2m 45s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/524/ : SUCCESS in 14s
- https://ci.intgdc.com/job/gooddata-ruby-tests-zuul/463/ : SUCCESS in 44m 39s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/200/ : FAILURE in 2m 26s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/530/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-ruby-tests-zuul/469/ : FAILURE in 43m 45s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/208/ : FAILURE in 1m 23s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/542/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-ruby-tests-zuul/481/ : FAILURE in 7m 11s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/220/ : SUCCESS in 1m 54s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/543/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-ruby-tests-zuul/482/ : FAILURE in 17s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/221/ : SUCCESS in 1m 53s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/546/ : SUCCESS in 14s
- https://ci.intgdc.com/job/gooddata-ruby-tests-zuul/485/ : FAILURE in 44m 59s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/224/ : FAILURE in 2m 05s

**Comment by vhtien** [comment]
ok to test

File: lib/gooddata/models/project.rb [hunk 1]
```diff
@@ -1128,7 +1128,10 @@ def objects(id, opts = {})
     def objects_export(objs, options = {})
       fail 'Nothing to migrate. You have to pass list of objects, ids or uris that you would like to migrate' if objs.nil?
       objs = Array(objs)
-      fail 'Nothing to migrate. The list you provided is empty' if objs.empty?
+      if objs.empty?
+        GoodData.logger.warn 'Nothing to migrate.'
+        return nil
```
Comment (panjan) [inline lib/gooddata/models/project.rb]: minor: redundant `nil`

File: lib/gooddata/models/project.rb [hunk 2]
```diff
@@ -1197,6 +1200,7 @@ def partial_md_export(objects, options = {})
       projects = options[:project]
       batch_size = options[:batch_size] || 10
       token = objects_export(objects)
+      return nil if token.nil?
```
Comment (panjan) [inline lib/gooddata/models/project.rb]: minor: redundant `nil`

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/548/ : SUCCESS in 15s
- https://ci.intgdc.com/job/gooddata-ruby-tests-zuul/487/ : SUCCESS in 43m 34s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/226/ : SUCCESS in 1m 17s

**Pull request merged**